app/controllers/v2/content_items_controller.rb

```diff
@@ -22,7 +22,7 @@ def linkables
     end
 
     def embedded
-      results = Queries::GetEmbeddedContent.new(
+      results = GetEmbeddedContentService.new(
```

Comment:
definitely a good improvement to have a service etc. but the name still confuses me... would `GetHostContentService` be more accurate now?

Reply:
Good plan 👍